scripts.js

```diff
@@ -0,0 +1,16 @@
+const hamburger = document.querySelector('.menu-bars');
+const dropdown = document.querySelector('.dropdown-menu');
+const close = document.querySelector('.close');
+const closeOption = document.querySelector('.mobile-menu-options');
+
+hamburger.addEventListener('click', () => {
+  dropdown.classList.remove('hide');
+});
+
+close.addEventListener('click', () => {
+  dropdown.classList.add('hide');
+});
+
+closeOption.addEventListener('click', () => {
+  dropdown.classList.add('hide');
+});
```

Comment:
Please, could you consider refactoring this statement? You assigned a listener to the container of the menu options rather than assigning one to each of the individual options. This leads to the malfunction illustrated below:

https://user-images.githubusercontent.com/20554583/184083574-12db0451-bce5-495d-8e63-8ea8ff8b181a.mp4

You might consider selecting a _nodeList_ of all the options using `querySelectorAll` and using the .forEach() function to assign the listener to each of them.